src/main/java/seedu/address/storage/PersonListStorage.java

```diff
@@ -0,0 +1,40 @@
+package seedu.address.storage;
+
+import java.io.IOException;
+import java.util.List;
+
+import seedu.address.model.person.ReadOnlyPerson;
+
+/**
+ * Represents a storage for a list of {@link seedu.address.model.person.Person}.
+ */
+public interface PersonListStorage {
+
+    /**
+     * Returns the file path of the data file.
+     */
+    public String getPersonListFilePath();
+
+    /**
+     * To be implemented
```

Comment:
you could still describe what it is supposed to do. that's the purpose of interfaces when designing an API